mkdocs.yml

```diff
@@ -0,0 +1,61 @@
+site_name: ColorHighlighter Documentation
+site_url: https://Monnoroch.github.io/ColorHighlighter
+repo_url: https://github.com/Monnoroch/ColorHighlighter
+edit_uri: tree/master/docs/src
+site_description: A color highlighting plugin for Sublime Text 2 and 3.
+
+pages:
+  - ColorHighlighter: index.md
+
+theme: material
+docs_dir: docs/src
+theme_dir: docs/theme
+
+markdown_extensions:
+  - markdown.extensions.toc:
+      slugify: !!python/name:pymdownx.slugs.gfm
+  - markdown.extensions.admonition:
+  - markdown.extensions.smarty:
+      smart_quotes: false
+  - pymdownx.betterem:
+  - markdown.extensions.attr_list:
+  - markdown.extensions.def_list:
+  - markdown.extensions.tables:
+  - markdown.extensions.abbr:
+  - pymdownx.extrarawhtml:
+  - pymdownx.superfences:
+  - pymdownx.highlight:
+      css_class: codehilite
+  - pymdownx.inlinehilite:
+  - pymdownx.magiclink:
+      repo_url_shortener: true
+      base_repo_url: https://github.com/facelessuser/ColorHighlighter
+  - pymdownx.tilde:
+  - pymdownx.caret:
+  - pymdownx.smartsymbols:
+  - pymdownx.emoji:
+      emoji_generator: !!python/name:pymdownx.emoji.to_png
+  - pymdownx.escapeall:
+      hardbreak: true
+      nbsp: true
+  - pymdownx.tasklist:
+      custom_checkbox: true
+  # - pymdownx.snippets:
```

Comment:
Why is this commented?

Reply:
Because I didn't add any snippets.  I normally put things like links in my snippets folder: https://github.com/facelessuser/Rummage/blob/master/docs/src/markdown/_snippets/links.md. And then link them at the bottom of a page: https://github.com/facelessuser/Rummage/blob/master/docs/src/markdown/usage.md.  I also use it to include other things, but currently the site is a single page, so I didn't see a reason to uncomment it.

Reply:
Perhaps, leave a TODO? Otherwise it looks like something is unfinished but not quite clear what.

Reply:
I can do that.  I wasn't sure what you'd actually use in the end.  I figured you'd restructure the docs and such and enable and/or disable what you didn't like.  I didn't want to presume what you'd end up using in the end.